diff --git a/node_modules/@bacons/apple-targets/build/target.js b/node_modules/@bacons/apple-targets/build/target.js
--- a/node_modules/@bacons/apple-targets/build/target.js
+++ b/node_modules/@bacons/apple-targets/build/target.js
@@ -104,6 +104,11 @@ exports.TARGET_REGISTRY = {
     "device-activity-monitor": {
         extensionPointIdentifier: "com.apple.deviceactivity.monitor-extension",
         frameworks: ["DeviceActivity"],
         displayName: "Device Activity Monitor",
     },
+    "device-activity-report": {
+        extensionPointIdentifier: "com.apple.deviceactivityui.report-extension",
+        frameworks: ["DeviceActivity", "SwiftUI"],
+        displayName: "Device Activity Report",
+    },
diff --git a/node_modules/@bacons/apple-targets/build/configuration-list.js b/node_modules/@bacons/apple-targets/build/configuration-list.js
--- a/node_modules/@bacons/apple-targets/build/configuration-list.js
+++ b/node_modules/@bacons/apple-targets/build/configuration-list.js
@@ -673,6 +673,7 @@ function createConfigurationList(props) {
         case "account-auth":
         case "bg-download":
         case "credentials-provider":
         case "device-activity-monitor":
+        case "device-activity-report":
         case "intent":
diff --git a/node_modules/@bacons/apple-targets/build/target.d.ts b/node_modules/@bacons/apple-targets/build/target.d.ts
--- a/node_modules/@bacons/apple-targets/build/target.d.ts
+++ b/node_modules/@bacons/apple-targets/build/target.d.ts
@@ -127,6 +127,11 @@ export declare const TARGET_REGISTRY: {
     readonly "device-activity-monitor": {
         readonly extensionPointIdentifier: "com.apple.deviceactivity.monitor-extension";
         readonly frameworks: readonly ["DeviceActivity"];
         readonly displayName: "Device Activity Monitor";
     };
+    readonly "device-activity-report": {
+        readonly extensionPointIdentifier: "com.apple.deviceactivityui.report-extension";
+        readonly frameworks: readonly ["DeviceActivity", "SwiftUI"];
+        readonly displayName: "Device Activity Report";
+    };
